smsapi/Api/Action/Base.cs

```diff
@@ -165,29 +118,48 @@ private bool isClientError(string code)
          * 999 Wewnętrzny błąd systemu
          * 201 Wewnętrzny błąd systemu
          */
-        private bool isHostError(string code)
+        private static bool IsHostError(string code)

```

Comment:
No need to be static, used only internally.

Reply:
I mean `static` does not stand against `private` access modifiers :) https://stackoverflow.com/a/135038

Reply:
Of course it's not, but this change was just unnecessary  :joy: